manuscript/index.qmd

```diff
@@ -51,24 +51,51 @@ We assume:
 
 ### Simulation model
 
-We use the generic model structure described above with a renewal process. To simulate noise in the infection process we assume additional Brownian noise for the effective reproduction number of XX.
+We use the generic model structure described above with a renewal process for reasons of convenience. The renewal equation is quite flexibie and makes minimal assumptions about the underlying infectious disease process. Although using the renewal equation to generate the data could be construed as leading to more favorable inference, all threee latent processes are able to generate the simulated data.
```

Comment:
```suggestion
We use the generic model structure described above with a renewal process as it represents (in its equivalence to an SEIR compartmental model) domain understanding of a model that can capture known infectious dynamics.
```

Reply:
I think we should add to the limitations: 

- We did not explore simulating data from all models.
- However, our choice (the renewal process model) reflects domain knowledge.
- This does mean that our simulated scenarios should be seen as a best case for the renewal process